tests/com/github/bazelbuild/rules_jvm_external/maven/OutdatedTest.java

```diff
@@ -438,4 +439,78 @@ public void worksWithCommonsVersions() throws IOException {
     assertThat(releaseInfo.releaseVersion, equalTo("20040616"));
     assertThat(releaseInfo.preReleaseVersion, is(nullValue()));
   }
+
+  @Test
+  public void shouldFindUpdatedVersionForBOMs() throws IOException {
+    Path artifactsFile = temp.newFile("outdated.artifacts").toPath();
+    Files.write(
+        artifactsFile, Arrays.asList("com.google.guava:guava:27.0-jre"), StandardCharsets.UTF_8);
+
+    Path bomsFile = temp.newFile("outdated.boms").toPath();
+    Files.write(
+        bomsFile, Arrays.asList("com.google.cloud:libraries-bom:26.0.0"), StandardCharsets.UTF_8);
+
+    Path repositoriesFile = temp.newFile("outdated.repositories").toPath();
+    Files.write(
+        repositoriesFile, Arrays.asList("https://repo1.maven.org/maven2"), StandardCharsets.UTF_8);
```

Comment:
Non-actionable comment: at some point, we should stop relying on Maven Central in our own tests since it breaks the hermeticity of the build.